src/libcore/slice/mod.rs

```diff
@@ -4152,6 +4152,23 @@ impl<'a, T> DoubleEndedIterator for Chunks<'a, T> {
             Some(snd)
         }
     }
+
+    #[inline]
+    fn nth_back(&mut self, n: usize) -> Option<Self::Item> {
+        let (end, overflow) = self.v.len().overflowing_sub(n * self.chunk_size);
```

Comment:
I think `n * self.chunk_size` could overflow here.

Reply:
Yes, that will. However, I was wondering what would the following test result:
```
let v: &[i32] = &[0, 1, 2, 3];
let mut c = v.chunks(5);
assert_eq!(c.nth_back(1_000_000_000usize), None);
```
Based on the implementation in this PR, this passes. Should it?? Or, should the result be:
```
assert_eq!(c.nth_back(1_000_000_000usize), &[0, 1, 2, 3]);
```

Reply:
It should have the same behavior as the default implementation of `nth_back`, which returns `None` there.